libs/common/src/lib/navbar/navbar.component.html

```diff
@@ -264,7 +264,17 @@
               <span class="nav-link-title">{{ 'navbar.nua-keys' | translate }}</span>
             </a>
           </div>
-          @if ((isBillingEnabled | async) && !noStripe) {
```

Comment:
Hello,
a minor thing: should this line be changed ?

Reply:
there was a conflict there, I thought I kept the main version but I probably did it wrong. I'll fix it to be like it is currently on stage